internal/net/grpc/metric/client_test.go

```diff
@@ -48,31 +48,15 @@ func TestNewClientHandler(t *testing.T) {
 		return nil
 	}
 	tests := []test{
-		// TODO test cases
-		/*
-		   {
-		       name: "test_case_1",
-		       args: args {
-		           opts: nil,
-		       },
-		       want: want{},
-		       checkFunc: defaultCheckFunc,
-		   },
-		*/
-
-		// TODO test cases
-		/*
-		   func() test {
-		       return test {
-		           name: "test_case_2",
-		           args: args {
-		           opts: nil,
-		           },
-		           want: want{},
-		           checkFunc: defaultCheckFunc,
-		       }
-		   }(),
-		*/
+		{
+			name: "return client handler",
+			args: args{
+				opts: nil,
+			},
+			want: want{
+				want: new(ClientHandler),
+			},
+		},
```

Comment:
please add the case of `opts is not nil`

Reply:
fixed